en/extras.xliff

```diff
@@ -0,0 +1,411 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file datatype="plaintext" source-language="en" target-language="en">
```

Comment:
Let's keep the `original` attribute. For reference, this is the the file in one of the add-ons

```
<file original="addon.qml" datatype="plaintext" source-language="en" target-language="en-US">
```